test/src/specs/consensus/insufficient_reward.rs

```diff
@@ -75,7 +75,7 @@ impl Spec for InsufficientReward {
     // export data
     // fn run(&self, net: &mut Net) {
     //     let node = &mut net.nodes[0];
-    //     let hashes = node.generate_blocks(100);
+    //     let hashes = mine(node, 100);
```

Comment:
I'm not sure about this comment block, is there any necessary keep this and event update the method here? or we can just delete this part if we don't run it anymore.

Reply:
Out of this PR.